gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
```

Comment:
We specifically avoid 4.10.x due to a fuckwitted by the gradle developers to allow it to mess with cached files, 4.9 was the last version which didn't have said change, there are 5.x and later versions which are also clean, but I need to properly validate newer gradle before we can switch to those.

Reply:
Reverted back to 4.9, local build successed